docs/reference/repositories-metering/repositories-metering.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[repositories-metering-apis]]
+== Repositories metering APIs
+
+experimental[]
+
+You can use the following APIs to retrieve repositories metrics.
+This is an API meant to be used by Elastic's commercial offerings.
+
+* <<get-repositories-metrics-api,Get repositories metrics>>
```

Comment:
Why call this metrics instead of metering?
Metrics sounds too general here, and is too similar to general "stats", what we wanted to avoid in the first place with the "metering" terminology.